cisc/lib.go

```diff
@@ -94,18 +96,16 @@ func (cfg *ciscConfig) proposeSendVoteUpdate(p *identity.Config) {
 	log.ErrFatal(cfg.ConfigUpdate())
 }
 
-// writes the ssh-keys to an 'authorized_keys'-file
+// writes the ssh-keys to an 'authorized_keys.cisc'-file
 func (cfg *ciscConfig) writeAuthorizedKeys(c *cli.Context) {
 	var keys []string
 	dir, _ := sshDirConfig(c)
-	authFile := dir + "/authorized_keys"
-	// Make backup
-	b, err := ioutil.ReadFile(authFile)
-	if err == nil {
-		err = ioutil.WriteFile(authFile+".back", b, 0600)
-		log.ErrFatal(err)
+	authFileOrig := filepath.Join(dir, "authorized_keys")
+	authFile := authFileOrig + ".cisc"
+	if _, err := os.Stat(authFileOrig); os.IsNotExist(err) {
+		log.Info("Making link from authorized_keys to authorized_keys.cisc")
+		os.Symlink(authFile, authFileOrig)
```

Comment:
Convenience - if you're using _only_ cisc, it will make a symlink for you, and it works out-of-the box. If you already have an `authorized_keys`, then it will not touch it.

Best would be if we could combine the two files somewhat - like copying `authorized_keys` to `authorized_keys.manual` and then merge that with the keys from CISC in the `authorized_keys`-file. But I'm afraid it'll be **too intelligent**. Ideal would be if `sshd` would allow for multiple `authorized_keys` - for that one would have to add

```
AuthorizedKeysFile ~/.ssh/authorized_keys ~/.ssh/authorized_keys.cisc
```

to `/etc/ssh/sshd_config`. I'll add that to the `README.md`